From e1843b86de99af4cfcb77bcb4ff0ecd98924671e Mon Sep 17 00:00:00 2001
From: Martin Sperl <kernel@martin.sperl.org>
Date: Mon, 25 Apr 2016 07:56:13 +0000
Subject: [PATCH 02/63] ARM: 2708: i2s: removed clock registers and added clock

Removed clock registers from reg and added clock to i2s devicetree node.

Signed-off-by: Martin Sperl <kernel@martin.sperl.org>
---
 arch/arm/boot/dts/bcm2708_common.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2708_common.dtsi b/arch/arm/boot/dts/bcm2708_common.dtsi
index f29558e..fbd04d7 100644
--- a/arch/arm/boot/dts/bcm2708_common.dtsi
+++ b/arch/arm/boot/dts/bcm2708_common.dtsi
@@ -161,8 +161,8 @@
 		i2s: i2s@7e203000 {
 			compatible = "brcm,bcm2835-i2s";
 			#sound-dai-cells = <0>;
-			reg = <0x7e203000 0x24>,
-			      <0x7e101098 0x08>;
+			reg = <0x7e203000 0x24>;
+			clocks = <&cprman BCM2835_CLOCK_PCM>;
 
 			dmas = <&dma 2>, <&dma 3>;
 			dma-names = "tx", "rx";
-- 
2.7.4

